source/regions/flags.rst

```diff
@@ -95,7 +95,7 @@ Conflicting Flags
 
 Sometimes, a certain location may have multiple overlapping regions with different values for the same flag. The following rules are used to determine which values are selected:
 
-* Regions will inherit the value of a flag from its parent, **if** the region did not have the flag set. (Note that the ``build`` flag is set implicitly with membership.)
+* Regions will inherit the value of a flag from its parent, **if** the region did not have the flag set (for the subject's associated region group, determined by the regions in the inheritance chain up to the child region). (Note that the ``build`` flag is set implicitly with membership.)
```

Comment:
This gets a bit messy here. Should probably just link to Priority and Inheritance for a more detailed explanation.